lib/db_utils.pm

```diff
@@ -67,15 +67,20 @@ sub influxdb_push_data {
     my ($url, $db, $org, $token, $data, %args) = @_;
     $args{quiet} //= 1;
     $args{proceed_on_failure} //= 0;
+    $args{timeout} //= 90;
     $data = build_influx_query($data);
+    # connectivity pre-check
+    if (script_run('curl --silent --insecure --fail ' . $url . '>/dev/null', timeout => $args{timeout}, die_on_timeout => 0)) {    # curl error
```

Comment:
```suggestion
    if (script_run('curl --silent --insecure --fail ' . $url . '>/dev/null', timeout => $args{timeout}, die_on_timeout => 0) != 0) {
```
I'd remove this comment as it's not useful and make the check for the return value explicit for better readability.

But those are two style suggestions, so take it or leave it